source/services/dataContracts/baseDataService/baseData.service.ts

```diff
@@ -41,16 +41,17 @@ export class BaseDataService<TDataType extends IBaseDomainObject, TSearchParams>
         return this._endpoint;
     }
 
-    private getItemEndpoint(id: number): string {
-        return this.endpoint + '/' + id.toString();
+    private getItemEndpoint(id: number, endpoint?: string): string {
```

Comment:
This is only used for the cloning functionality. The clone provides getList, getDetail, create, update, and delete functions that just call the base with an updated endpoint generated when the clone is created.